### ISSUE TITLE
Fenix 1.7.3 - won't boot sd card anymore

### ISSUE THREAD

**Issue by hwspeedy** [khadas/fenix#363]
Somewhere between fenix 1.5.2 and 1.7.3 the boot sd card first is broken - probably mainline uboot missing fixes...

tested on wim3 - both ubuntu 2404 and debian12.

**Comment by lockheed** [comment]
Exactly. I just found the generated images do not even contain partition table. The resulting img file contains just noise.

**Comment by lockheed** [comment]
@numbqq could you comment on this, please? Because as it is now, the whole script seems completely unusable.

**Comment by lockheed** [comment]
Ok, I found in some random post a suggestion that perhaps those fenix images should be flashed through OOWOW (such critical point should be in the readme, IMO). So I did that, but none of those would boot, anyway.

**Comment by viraniac** [comment]
@lockheed Could you please share more information?

1. Do you have anything flashed to emmc? 
2. Can you share information about the sdcard you are using?
3. Also as you have mentioned that resulting image only contains noise and no partition table, could you please share the image that you are trying to flash?
4. Is the image self generated or downloaded from Khadas website? If self generated, could you please share the parameters (output of `make info`) you have used to generate the image?
